astropy/convolution/convolve.py

```diff
@@ -312,6 +312,9 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0, crop=True,
         256.
     quiet : bool, optional
         Silence warning message about NaN interpolation
+    normalized_epsilon: float, optional
+        The upper limit on the deviation of the kernel sum with respect to unity.
+        Will be checked only if ``normalize_kernel`` is False. Default is "1e-8".
```

Comment:
I think this should mention somewhere that it is a relative tolerance. Since elsewhere we often use ``rtol`` for relative tolerance, I wonder if ``normalization_rtol`` might be a better argument name? (no strong opinions though).